advice_animal/cli.py

```diff
@@ -147,30 +145,52 @@ def test(ctx, show_exception):
     return rv
 
 
+def find_python_projects(path: Path) -> list[Path]:
+    """
+    Find every project in a mono-repo (or not).
+    """
+    projects = []
+
+    for dirpath, dirnames, filenames in os.walk(path):
```

Comment:
Looks like the filtering of dirs that start with a dot is on us, as is handling gitignore.  There may be a simpler way to do this, but trailrunner includes an implementation using pathspec.

See `def gitignore` (which needs `def project_root`) and is used at https://github.com/omnilib/trailrunner/blob/f1693b383fa7afddfd21d43ec2db2711197f78e2/trailrunner/core.py#L177